Composer/packages/client/src/pages/design/exportSkillModal/content/SelectProfile.tsx

```diff
@@ -0,0 +1,208 @@
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import { PublishTarget, SkillManifestFile } from '@bfc/shared';
+import formatMessage from 'format-message';
+import React, { useEffect, useMemo, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { TooltipHost } from 'office-ui-fabric-react/lib/Tooltip';
+import { Icon } from 'office-ui-fabric-react/lib/Icon';
+import { Dropdown, IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+
+import { botDisplayNameState, dispatcherState, settingsState, skillManifestsState } from '../../../../recoilModel';
+import { CreatePublishProfileDialog } from '../../../botProject/CreatePublishProfileDialog';
+import { iconStyle } from '../../../botProject/runtime-settings/style';
+import { ContentProps, VERSION_REGEX } from '../constants';
+
+const styles = {
+  container: css`
+    height: 350px;
+    overflow: auto;
+  `,
+};
+
+const onRenderLabel = (props) => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'row',
+        marginBottom: '5px',
+      }}
+    >
+      <div
+        style={{
+          marginRight: '5px',
+          fontWeight: 600,
+          fontSize: '14px',
+        }}
+      >
+        <div
+          style={{
+            padding: '0 5px',
+          }}
+        >
+          {props.label}
+        </div>
+      </div>
+      <TooltipHost content={props.ariaLabel}>
+        <Icon iconName="Info" styles={iconStyle(props.required)} />
+      </TooltipHost>
+    </div>
+  );
+};
+
+export const getManifestId = (
+  botName: string,
+  skillManifests: SkillManifestFile[],
+  { content: { $schema } = {} }: Partial<SkillManifestFile>
+): string => {
+  const [version] = VERSION_REGEX.exec($schema) || [''];
+
+  let fileId = version ? `${botName}-${version.replace(/\./g, '-')}-manifest` : `${botName}-manifest`;
+  let i = -1;
+
+  while (skillManifests.some(({ id }) => id === fileId) && i < skillManifests.length) {
+    if (i < 0) {
+      fileId = fileId.concat(`-${++i}`);
+    } else {
+      fileId = fileId.substr(0, fileId.lastIndexOf('-')).concat(`-${++i}`);
+    }
+  }
+
+  return fileId;
+};
+
+export const SelectProfile: React.FC<ContentProps> = ({ manifest, setSkillManifest, value, onChange, projectId }) => {
+  const [publishingTargets, setPublishingTargets] = useState<PublishTarget[]>([]);
+  const [currentTarget, setCurrentTarget] = useState<PublishTarget>();
+  const { updateCurrentTarget } = useRecoilValue(dispatcherState);
+  const settings = useRecoilValue(settingsState(projectId));
+  const [endpointUrl, setEndpointUrl] = useState<string>();
+  const [appId, setAppId] = useState<string>();
+  const { id, content } = manifest;
+  const botName = useRecoilValue(botDisplayNameState(projectId));
+  const skillManifests = useRecoilValue(skillManifestsState(projectId));
+
+  const { ...rest } = content;
```

Comment:
you can use content itself instead of ...rest

Reply:
use itself